data/Chinwenma.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "Chinwenma Okorie",
+  "bio": " **FrontEnd Developer** | Open Source enthusiast | Blogger | Tech enthusiast",
+  "tags": [
+    "Open Source",
+    "ReactJs",
+    "Javascript",
+    "Tailwind",
+     "Html5",
+      "CSS3"
+  ],
+    "socials": [
+    { "icon": "FaTwitter", 
+      "url": "https://twitter.com/Ethel_Omma"
+    }
+    { "icon": "FaGithub",
+      "url": "https://github.com/Chinwenma"
+    }
+     { "icon": "FaLinkedin",
+       "url": "https://www.linkedin.com/in/chinwenma-okorie-5414511a6?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3Bib%2FYeFWESsiAGxMfYOHqFQ%3D%3D"
+     }
+  ],
+
```

You've unfortunately removed all the required commas

```suggestion
    "socials": [
    { "icon": "FaTwitter", 
      "url": "https://twitter.com/Ethel_Omma"
    },
    { "icon": "FaGithub",
      "url": "https://github.com/Chinwenma"
    },
     { "icon": "FaLinkedin",
       "url": "https://www.linkedin.com/in/chinwenma-okorie-5414511a6?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3Bib%2FYeFWESsiAGxMfYOHqFQ%3D%3D"
     }
  ],

```